res/controllers/Reloop Terminal Mix 2-4.js

```diff
@@ -121,6 +121,11 @@ TerminalMix.samplerVolume = function (channel, control, value) {
     }
 }
 
+TerminalMix.pitchSlider = function (channel, control, value, status, group) {
+	// invert pitch slider (down=faster) so it matches the labels on controller
+	engine.setValue(group,"rate",-script.midiPitch(control, value, status));
+}
+
```

Comment:
Oh sorry, my mistake. I forgot pitch bend signals use both the second and third byte as the LSB and MSB.